src/Problems/Problem14.java

```diff
@@ -0,0 +1,39 @@
+package Problems;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Problem14 {
+    //TODO: minimum amount of coins for 33 (10+10+10+3 = 4 coins)
+
+    public static void numbers (int number) {
+        ArrayList<Integer> list = new ArrayList<>();
+        ArrayList<Integer> coins = new ArrayList<Integer>();
+        coins.add(1);
+        coins.add(3);
+        coins.add(10);
+        coins.add(25);
+        coins.add(50);
```

Comment:
Can you try this variant?
ArrayList<Integer> numbers = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5, 6));